lib/job/arActorExportJob.class.php

```diff
@@ -71,8 +71,33 @@ protected function doExport($path)
                 return;
             }
 
-            $this->exportResource($resource, $path);
+            $this->csvActorExport($path, $resource);
             $this->logExportProgress();
         }
     }
+
+    protected function csvActorExport($path, $resource)
+    {
+        $configuration = ProjectConfiguration::getApplicationConfiguration('qubit', 'prod', false);
+        $this->context = sfContext::createInstance($configuration);
+
+        // Prepare CSV exporter
+        $writer = new csvActorExport($path);
+        $writer->setOptions(['relations' => true]);
+
+        // Export actors and, optionally, related data
+        $itemsExported = 0;
+
+        $cultures = array_keys(DefaultTranslationLinksComponent::getOtherCulturesAvailable($resource->actorI18ns, 'authorizedFormOfName', $resource->getAuthorizedFormOfName(['sourceCulture' => true])));
+
+        // Write row to file and initialize row
+        foreach ($cultures as $culture) {
+            $actor = QubitActor::getById($resource->id);
+            $this->context->getUser()->setCulture($culture);
+
+            $writer->exportResource($actor);
+
+            ++$itemsExported;
```

Comment:
Are we using this count or logging this somewhere? If not, we can skip this variable entirely.

Reply:
`$itemsExported` is used in logs to show how many records were exported. although, with my new implementation, it's reporting number of rows exported, so i could move it out of the loop to accurrately represent items exported? (but descriptions are also reporting 2 records)